yar/yar.php

```diff
@@ -47,7 +47,7 @@ class Yar_Server
      * @param object $obj An Object, all public methods of its will be registered as RPC services.
      * @link https://secure.php.net/manual/en/yar-server.construct.php
      */
-    final public function __construct($obj, $protocol = null) {}
+    final public function __construct($obj) {}
```

Comment:
There is no second param anywhere
checked in phpdocs and in the repo: https://github.com/laruence/yar/search?q=Yar_Server

Reply:
Tests seem no to agree: https://github.com/JetBrains/phpstorm-stubs/pull/1110/checks?check_run_id=2318581258#step:5:1372

Are tests made to pass for the content or pass because of the content 🤔 

I did not find any source having a second param, checked 8 pages: https://github.com/search?l=PHP&p=1&q=new+Yar_Server%28&type=Code

Reply:
Tests compare data that is returned by reflection and data that is set in stubs. I've just tried to run next code with installed yar extension
```
<?php
foreach(((new ReflectionClass(Yar_Server::class))->getMethod('__construct'))->getParameters() as $parameter) {
    var_dump($parameter);
}
```
and result is next
<img width="380" alt="Screenshot 2021-04-11 at 23 03 28" src="https://user-images.githubusercontent.com/37301285/114319418-31afec80-9b1a-11eb-8fd8-1135277ac5cc.png">

I'm not a guru in C language and can't say right now where exactly this parameter is come from but I trust to reflection :)